config-overrides.js

```diff
@@ -11,6 +12,7 @@ module.exports = {
             include: [
                 path.resolve(srcRoot, 'assets/icons'),
                 path.resolve(uiKitRoot, 'assets/icons'),
+                uiKitIconsRoot,
```

Comment:
I don't know why it was resolved to 'assets/icons', so I left it as it was, but added path to uikit icons